dev/bots/test.dart

```diff
@@ -130,15 +129,6 @@ Future<Null> _analyzeRepo() async {
   print('${bold}DONE: Analysis successful.$reset');
 }
 
-Future<Null> _runTestsDart2() async {
```

Comment:
We should probably run a shard with `--no-preview-dart-2` now.

Reply:
Although I guess that would preclude us from big cleanups, like new/const removal... nevermind 😄

Reply:
We want to have that temporarily, but when I tried it I discovered that it does not fully work. Some of the stuff that `_runTests` invokes did not understand `--no-preview-dart-2`. In any case we most likely are not gonna support `--no-preview-dart-2` for long. If Flutter framework starts using features like reified generic methods or optional new/const, there is no way to run this on Dart 1.